<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forseti v. 1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Processes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="5">
         <widget class="QTreeWidget" name="treeWidget">
          <property name="columnCount">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dumpButton">
          <property name="text">
           <string>Dump Process</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="dumpAllButton">
          <property name="text">
           <string>Dump All Processes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="saveProcButton">
          <property name="text">
           <string>Save Proc List</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="scanCryptoButton">
          <property name="text">
           <string>Scan for Crypto</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Memory Map</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QTreeWidget" name="treeWidget_2">
          <property name="columnCount">
           <number>6</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">4</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">5</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">6</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dumpRegionButton">
          <property name="text">
           <string>Dump Region</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="savemapButton">
          <property name="text">
           <string>Save Map</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Threads</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget_5">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Opened Files</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QTreeWidget" name="treeWidget_3">
          <property name="columnCount">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="saveOFilesListButton">
          <property name="text">
           <string>SaveList</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="refrOFilesButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Process Stats</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QTreeWidget" name="treeWidget_4">
          <property name="columnCount">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="refrStatsButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="saveStatsButton">
          <property name="text">
           <string>Save Stats</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Network Activity</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0" colspan="2">
         <widget class="QTreeWidget" name="treeWidget_6">
          <property name="columnCount">
           <number>12</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">4</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">5</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">6</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">7</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">8</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">9</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">10</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">11</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">12</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="rfrshNetButton">
          <property name="text">
           <string>Refresh List</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="saveNtwButton">
          <property name="text">
           <string>Save List</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_Version"/>
    <addaction name="action_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Version">
   <property name="text">
    <string>&amp;Version</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
